ui/src/index.tsx

```diff
@@ -1,17 +1,36 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import React from "react"
+import ReactDOM from "react-dom"
+import "./index.css"
+import App from "./App"
+import * as serviceWorker from "./serviceWorker"
+import { Hub } from "./api"
+import { KindStore } from "./store/kind"
+import { CatalogStore } from "./store/catalog"
+import { CategoryStore } from "./store/category"
+import { ResourceStore } from "./store/resources"
+
+const store = ResourceStore.create(
+  {},
+  {
+    api: new Hub(),
+    kindStore: KindStore.create({}),
+    catalogStore: CatalogStore.create({}),
+    categoryStore: CategoryStore.create({}, { api: new Hub() }),
+  }
+)
```

Comment:
```suggestion
const api = new Hub()

const store = ResourceStore.create(
  {},
  {
    api,
    kindStore: KindStore.create({}),
    catalogStore: CatalogStore.create({}),
    categoryStore: CategoryStore.create({}, { api }),
  }
)
```

Reply:
@vinamra28 thanks for suggestions will change :smiley: